### PR TITLE
Add More Commands to OctoprintClient

### PR DESCRIPTION
**Pull request by AshtonKem**
Create the ability to:

- Connect & Disconnect
- Set bed temperature
- Set tool temperature

Bump version number to 0.1.10

### CODE REVIEW

File: pyoctoprintapi/octoprint_client.py [hunk 1]
```diff
@@ -98,4 +100,34 @@ async def pause_job(self) -> None:
...
         _LOGGER.debug("Sending resume job command")
-        await self._api.issue_job_command(JOB_COMMAND_PAUSE, JOB_COMMAND_PAUSE_RESUME)
\ No newline at end of file
+        await self._api.issue_job_command(JOB_COMMAND_PAUSE, JOB_COMMAND_PAUSE_RESUME)
+
+    async def disconnect(self) -> None:
+        _LOGGER.debug("Disconnecting from printer")
+        await self._api.issue_connection_command("disconnect")
+
+    async def connect(self, port: Optional[str] = None, baud_rate: int = None, printer_profile: str = None, save: bool = None, auto_connect: bool = None) -> None:
```
Comment (rfleming71) [inline pyoctoprintapi/octoprint_client.py]: Should all this parameters be marked optional seeing as None is the default
```suggestion
    async def connect(self, port: Optional[str] = None, baud_rate: Optional[int] = None, printer_profile: Optional[str] = None, save: Optional[bool] = None, auto_connect: Optional[bool] = None) -> None:
```

File: pyoctoprintapi/octoprint_client.py [hunk 2]
```diff
@@ -98,4 +100,34 @@ async def pause_job(self) -> None:
 
     async def resume_job(self) -> None:
         _LOGGER.debug("Sending resume job command")
-        await self._api.issue_job_command(JOB_COMMAND_PAUSE, JOB_COMMAND_PAUSE_RESUME)
\ No newline at end of file
+        await self._api.issue_job_command(JOB_COMMAND_PAUSE, JOB_COMMAND_PAUSE_RESUME)
+
+    async def disconnect(self) -> None:
+        _LOGGER.debug("Disconnecting from printer")
+        await self._api.issue_connection_command("disconnect")
```
Comment (rfleming71) [inline pyoctoprintapi/octoprint_client.py]: Can the command string be moved into the consts.py file?
```suggestion
        await self._api.issue_connection_command(CONNECTION_COMMAND_DISCONNECT)
```

File: pyoctoprintapi/octoprint_client.py [hunk 3]
```diff
@@ -98,4 +100,34 @@ async def pause_job(self) -> None:
...
+
+    async def connect(self, port: Optional[str] = None, baud_rate: int = None, printer_profile: str = None, save: bool = None, auto_connect: bool = None) -> None:
+        _LOGGER.debug("Connecting to printer")
+        await self._api.issue_connection_command("connect", port=port, baudrate=baud_rate, printerprofile=printer_profile, save=save, autoconnect=auto_connect)
```
Comment (rfleming71) [inline pyoctoprintapi/octoprint_client.py]: Can the command string be moved into the consts.py file?
```suggestion
        await self._api.issue_connection_command(CONNECTION_COMMAND_CONNECT, port=port, baudrate=baud_rate, printerprofile=printer_profile, save=save, autoconnect=auto_connect)
```

**Review by rfleming71 — CHANGES_REQUESTED** [review]
Couple of small style requests, but everything works great for me. Thanks for the PR!

File: pyoctoprintapi/octoprint_client.py (hunk 1 above)
Comment (AshtonKem) [reply pyoctoprintapi/octoprint_client.py]: Hah! I did this originally, but thought it was wrong. I stopped using python before type hinting became a thing.

**Review by rfleming71 — APPROVED** [review]

**Pull request merged**